docs/simple_examples.md

```diff
@@ -0,0 +1,45 @@
+## Create first Virtual Machines
+
+1. Generate the product before installation:
+    ```
+    ./mdbci generate-product-repositories --product mariadb
+    ```
+   [Read more about generate-product-repositories command](commands/generate-product-repositories.md)
```

Comment:
Do not use such long reference names. Please also add dots to the end of sentences